### DISCUSSION TITLE
Create a tiptap to html / pdf converter in Java

### DISCUSSION THREAD

**Discussion by JamesStandbridge** [ueberdosis/tiptap#5332] [General Questions] [answered]
Hello,

As part of a project, I'm going to have to create a converter from json tiptap format to pdf (so html I imagine) using Java Spring. I don't think this is trivial at all, especially with extension management. It may even be impossible. Do you have any information, feedback or anything else on the subject?

**Comment by nperez0111** [comment, marked as answer]
Would definitely recommend transforming to html first and then getting that to pdf. 

We are working on a service for exporting tiptap json but do not yet support PDF (we will soon though) https://tiptap.dev/docs/editor/extensions/functionality/export
